stablehlo/transforms/Passes.h

```diff
@@ -86,6 +86,20 @@ std::unique_ptr<OperationPass<ModuleOp>> createStablehloRefineArgumentsPass(
 // the current version in order to legalize to StableHLO.
 void createStablehloDeserializePipeline(OpPassManager &pm);
 
+// Creates a pipeline of StableHLO-specific MLIR passes to refine the "main"
+// function's arguments and propagates new shapes throughout the program
+// argument types and shapes within an MLIR module. The main function is either
+// a function with name "main", if there are multiple functions, or the single
+// function within the module.
+//
+// This pipeline focuses on:
+//   1. Refining function argument types based on provided `refinedTypes`.
+//   2. Refining shape information of operations within functions.
+//   3. Replaces dynamic StableHLO ops with the corresponding static
+//   counterparts if applicable.
```

Comment:
I think this is a good description, but consider opening with a direct description of the goal: to remove dynamism from the program.

Reply:
done